crates/vm/levm/src/vm.rs

```diff
@@ -106,7 +109,7 @@ impl VM {
                     calldata.clone(),
                     false,
                     env.gas_limit,
-                    TX_BASE_COST,
+                    U256::zero(),
```

Comment:
Why this?

Reply:
Because `TX_BASE_COST` is added in the intrinsic gas function.